lib/cequel/record/tasks.rb

```diff
@@ -26,6 +26,7 @@
     project_root = defined?(Rails) ? Rails.root : Dir.pwd
     models_dir_path = "#{File.expand_path('app/models', project_root)}/"
     model_files = Dir.glob(File.join(models_dir_path, '**', '*.rb'))
+    model_files.reject! {|file| file =~ /^#{File.join(models_dir_path, 'concerns')}/}
```

Comment:
@islue is this necessary? I think the code below should already handle any bad situation that might pop up…